pkg/runtime/pubsub/bulksubscribe_events.go

```diff
@@ -0,0 +1,72 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package pubsub
+
+import (
+	"github.com/google/uuid"
+
+	"github.com/dapr/kit/logger"
+
```

Comment:
Nit: no newline here

Reply:
I tried incorporating it, but it is failing in linting:
pkg/runtime/pubsub/bulksubscribe_events.go:17  goimports   File is not `goimports`-ed with -local github.com/dapr/